iocuddle-sev/build.rs

```diff
@@ -13,9 +13,19 @@
 // limitations under the License.
 
 fn main() {
+    use std::fs;
     use std::path::Path;
 
     if Path::new("/dev/sev").exists() {
+        let sev_parameter = fs::read_to_string("/sys/module/kvm_amd/parameters/sev")
+            .expect("Failed to read SEV kernel parameter value");
+        let sev_parameter = sev_parameter.trim();
+        if sev_parameter != "1" {
+            panic!(
+                "AMD SEV is not enabled in this kernel - expected these kernel parameters: \
+                   mem_encrypt=on, kvm_amd.sev=1"
+            );
+        }
```

Comment:
In hindsight, I do not think this is a useful build-time check. I am inclined to drop this commit. I think this kind of thing would be a useful runtime check that would ideally occur long before the flow of control can reach an area where an ioctl call is possible.

Reply:
Per the meeting this morning, this kind of check would be more appropriate with a SEV flavored version of `sgx-show`. Opened #205